src/libaudtag/id3/id3-common.cc

```diff
@@ -169,6 +169,24 @@ void id3_decode_comment (Tuple & tuple, const char * data, int size)
         tuple.set_str (Tuple::Comment, value);
 }
 
+void id3_decode_lyrics (Tuple & tuple, const char * data, int size)
```

Comment:
Looks like an exact copy of `id3_decode_comment`, except for changing `Tuple::Comment` to `Tuple::Lyrics`. Please combine to avoid duplicating code.

Reply:
Changed to 'id3_associate_memo'